presto-main/src/main/java/com/facebook/presto/server/QueryStateInfoResource.java

```diff
@@ -86,42 +88,60 @@ public QueryStateInfoResource(
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public void getQueryStateInfos(@QueryParam("user") String user,
-            @QueryParam("includeLocalQueryOnly") @DefaultValue("false") boolean isIncludeLocalQueryOnly,
+    public void getQueryStateInfos(
+            @QueryParam("user") String user,
+            @QueryParam("includeLocalQueryOnly") @DefaultValue("false") boolean includeLocalQueryOnly,
+            @QueryParam("includeAllQueries") @DefaultValue("false") boolean includeAllQueries,
+            @QueryParam("includeAllQueryProgressStats") @DefaultValue("false") boolean includeAllQueryProgressStats,
+            @QueryParam("excludeResourceGroupPathInfo") @DefaultValue("false") boolean excludeResourceGroupPathInfo,
+            @QueryParam("limit") Integer limitFilter,
+            @QueryParam("queryTextSizeLimit") Integer queryTextSizeLimit,
```

Comment:
For disaggregated coordinator clusters, this endpoint sends requests to DistributedQueryStateInfoResource, which means we need to implement this filters in the distributed version of this endpoint as well. Otherwise this will not behave correctly in case of disaggregated coordinator clusters.

Reply:
This should not be needed since the way scatter/gather happens in the RM, it just passes the URI (which includes the params) as it is and hence we don't need to make any changes.